src/main/drivers/serial_uart_hal.c

```diff
@@ -233,10 +247,14 @@ void uartSetMode(serialPort_t *instance, portMode_e mode)
     uartReconfigure(uartPort);
 }
 
+#ifdef USE_DMA
 void uartTryStartTxDMA(uartPort_t *s)
 {
+    uint16_t size = 0;
```

Comment:
Is there a reason that these have been moved up? In my opinion, declaring variables in the closest scope possible makes the code easier to read.

Reply:
I don't remember why I did this. They are back into the original position.